aten/src/ATen/cpu/vec/vec512/vec512_double.h

```diff
@@ -83,27 +83,15 @@ template <> class Vectorized<double> {
     if (count == size())
       return _mm512_loadu_pd(reinterpret_cast<const double*>(ptr));
 
-
-    __at_align__ double tmp_values[size()];
-    // Ensure uninitialized memory does not change the output value See https://github.com/pytorch/pytorch/issues/32502
-    // for more details. We do not initialize arrays to zero using "={0}" because gcc would compile it to two
-    // instructions while a loop would be compiled to one instruction.
-    for (const auto i : c10::irange(size())) {
-      tmp_values[i] = 0.0;
-    }
-    std::memcpy(
-        tmp_values,
-        reinterpret_cast<const double*>(ptr),
-        count * sizeof(double));
-    return _mm512_load_pd(tmp_values);
+    __mmask8 mask = (1 << count) - 1;
```

Comment:
also use 1ULL?

Reply:
Thanks for the reminder. Seems I missed it, raised another PR for it: https://github.com/pytorch/pytorch/pull/117021